config/webpack.config.js

```diff
@@ -8,7 +8,7 @@ const WebpackChunkHash = require('webpack-chunk-hash');
 
 const isProd = process.env.NODE_ENV === "production";
 const { NODE_ENV, APP_ID, API_KEY, INDEX_NAME } = process.env;
-const SCRIPT_SRC = NODE_ENV === 'production' ? 'https://cdn.jsdelivr.net/npm/search-insights@0.0.15/dist/search-insights.min.js' : 'http://localhost:8080/search-insights.min.js';
+const SCRIPT_SRC = NODE_ENV === 'production' ? 'https://cdn.jsdelivr.net/npm/search-insights@~0.0.16/dist/search-insights.min.js' : 'http://localhost:8080/search-insights.min.js';
```

Comment:
Why we don't stick with a fixed version?

Reply:
cc @tkrugg

Reply:
Sorry I missed your comment.
I guess that would imply manually updating the examples at every release, which would make sense between major versions with breaking changes but not for minors imho.
Would you be in favour of fixing it?

Reply:
Yes at least we ensure that every time we run the example we always have the same version.